### PR TITLE
Create Manager Based Cartpole Vision Example Environments

### PR DESCRIPTION
**Pull request by glvov-bdai**
# Description

Adds manager based cartpole vision example environments. 

Also uses the`convert_perspective_depth_to_orthogonal_depth ` functionality introduced in #976 , contains a duplicate copy of the method and test for completeness of the PR. Will be synced with main to remove this duplicate copy once #976 is merged into main, or #976 will be synced with main if this PR is merged first

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Screenshots
image

## Checklist

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

## Testing

Training converged in both RGB and Depth at similar rates to the direct environments

### CODE REVIEW

**Comment by jsmith-bdai** [comment]
Will review this soon!

Can you update the base branch to be the one from https://github.com/isaac-sim/IsaacLab/pull/976 so only new changes are reviewed here :pray:. Can help out with this if it doesn't make sense

File: source/extensions/omni.isaac.lab/docs/CHANGELOG.rst
Comment (jsmith-bdai) [inline source/extensions/omni.isaac.lab/docs/CHANGELOG.rst:1]: Make sure to update this one after the other merges!

File: source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py
Comment (jsmith-bdai) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py]: NIT: I would hold back on the highly recommended for training performance
```suggestion
        normalize: Set to True to normalize images.
```

File: source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py
Comment (jsmith-bdai) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py:187]: NIT: why default to tiled camera?
```suggestion
    sensor_cfg: SceneEntityCfg = SceneEntityCfg("tiled_camera"),
```

File: source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/cartpole_camera_env_cfg.py
Comment (jsmith-bdai) [inline source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/cartpole_camera_env_cfg.py]: Why have both RGB and Depth in the same Observation Cfg rather than 2 separate classes?

**Review by jsmith-bdai — APPROVED** [review]
Approving to unblock, looks good!

File: source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/cartpole_camera_env_cfg.py
Comment (glvov-bdai) [reply source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/cartpole_camera_env_cfg.py]: Split it up into 2 for better readability

initially only had it as one for the sake of brevity

File: source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py
Comment (glvov-bdai) [reply source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py:187]: Default to tiled as it is the most performant camera that can handle multiple dynamic objects, seemed like the obvious choice at the time but down to remove the default arg

File: docs/source/overview/environments.rst
Comment (Dhoeller19) [inline docs/source/overview/environments.rst:60]: Can you add blank lines between the various links?
This is a new addition we made in 1.2 to make the table look nicer

File: source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py
Comment (Dhoeller19) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py]: Is there a reason this is `True` by default?
Since this is mostly used in RL, the policy will probably handle both cases like a champ.
In that case I would opt for the route with the lowest compute by default.

**Review by Dhoeller19 — APPROVED** [review]
Thanks @garylvov!

File: source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py
Comment (glvov-bdai) [reply source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py]: I originally set this to `True` by default because I was thinking about using point cloud observation space, in which case, `unproject_depth` requires an "orthogonal" depth image.

However, since this method doesn't do any point cloud stuff, only depth image, I agree that the RL policy should easily adapt to perspective depth, and that it's better to have a lower compute default. Good spot!

TLDR: changing default to `False`

**Comment by glvov-bdai** [comment]
@jsmith-bdai ready to merge

File: source/extensions/omni.isaac.lab_tasks/docs/CHANGELOG.rst
Comment (jsmith-bdai) [inline source/extensions/omni.isaac.lab_tasks/docs/CHANGELOG.rst]:
```suggestion
0.10.6 (2024-09-25)
```

**Pull request merged**
